### ISSUE TITLE
About the training loss turning to NaN.

### ISSUE THREAD

**Issue by naturezhanghn** [xuboming8/DSTNet#6]
When training on Gopro with the current settings, I encounter an issue where the loss turns into nan. How can this be resolved?

**Comment by xuboming8** [comment]
You can reload the previous model with a correct loss to solve this problem.

**Comment by naturezhanghn** [comment]
I train on my own data and the loss is always nan.

**Comment by xuboming8** [comment]
Due to the instability of dynamic convolution, there are three methods you can try：
1. use gradient clip method: torch.nn.utils.clip_grad_norm_(self.net_g.parameters(), 0.01)
2. reload the previous model with a correct loss
3. reduce learning rate
